src/applications/disability-benefits/all-claims/pages/adaptiveBenefits.js

```diff
@@ -0,0 +1,50 @@
+import {
+  benefitDescription,
+  doubleAllowanceAlert,
+} from '../content/adaptiveBenefits';
+
+export const uiSchema = {
+  'ui:title': 'Automobile allowance and adaptive benefits',
+  'ui:description': benefitDescription,
+  'view:modifyingHome': {
+    'ui:title': 'Do you need help buying or modifying your home?',
+    'ui:widget': 'yesNo',
+  },
+  'view:modifyingCar': {
+    'ui:title': 'Do you need help buying or modifying your car?',
+    'ui:widget': 'yesNo',
+  },
+  'view:alreadyClaimedVehicleAllowance': {
+    'ui:title': 'Have you ever been granted an automobile allowance?',
+    'ui:widget': 'yesNo',
+    'ui:options': {
+      expandUnder: 'view:modifyingCar',
```

Comment:
Alternatively, both this and `view:doubleAllowanceAlert` could be nested within another `view:needsCarHelp` property that had the `expandUnder` option applied to it. Then the `doubleAllowanceAlert` would only need a `hideIf` and the `expandUnder` wouldn't need to be repeated across both. But yeah, not sure either approach makes a huge difference. Might be a bit more clear.

Reply:
True. I was trying to keep it flatter, but since these are all `view:` properties and will be capturing no data, I can nest however deep we like without any fear of trouble later.